default/stringtables/en.txt

```diff
@@ -157,11 +157,10 @@ Fleet
 # these are applied to the %1% style placeholders according the order in which the
 # parameters are specified in the effect
 NEW_ASTEROIDS_NAME
-%1% %2% remnants
+%1% %2% %3% remnants
```

Comment:
They are not, without an exclusion variable you would have a gap in planet numbering.  In the event of multiple asteroid belts, they would all have the same name.

If duplicate asteroid names are not an issue, I can drop the valueref and change the default naming to exclude asteroids.